hivelvet-backend/app/src/Actions/Rooms/Add.php

```diff
@@ -83,8 +83,12 @@ public function save($f3, $params): void
                         $this->logger->error($errorMessage, ['error' => 'Room Link already exists']);
                         $this->renderJson(['errors' => ['short_link' => 'Room link already exists']], ResponseCode::HTTP_PRECONDITION_FAILED);
                     } else {
+                        while ($checkRoom->meetingIdExists($room->meeting_id)) {
+                            $room->meeting_id = bin2hex(openssl_random_pseudo_bytes(8));
```

Comment:
We should have a "unique" meeting_id per room. Recordings of the same room can be fetched using that same id.